autogpt_platform/docker-compose.platform.yml

```diff
@@ -4,17 +4,18 @@ services:
       context: ../
       dockerfile: autogpt_platform/backend/Dockerfile
       target: server
-    command: ["sh", "-c", "poetry run prisma migrate deploy"]
+    command: ["sh", "-c", "poetry run migrate_db"]
     develop:
       watch:
         - path: ./
-          target: autogpt_platform/backend/migrate
+          target: autogpt_platform/backend/migrations
           action: rebuild
     depends_on:
       db:
         condition: service_healthy
     environment:
       - DATABASE_URL=postgresql://postgres:your-super-secret-and-long-postgres-password@db:5432/postgres?connect_timeout=60&schema=platform
+      - DIRECT_DATABASE_URL=postgresql://postgres:your-super-secret-and-long-postgres-password@db:5432/postgres?connect_timeout=60
```

Comment:
I think we can use this URL as  DATABASE_URL, and simply define the schema on db.py, where the only prisma.connect exist.

Reply:
Just checked https://www.prisma.io/docs/orm/prisma-schema/data-model/multi-schema
We don't even need to change it, you can simply use `DATABASE_URL`